.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Build master document
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build_dict:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: setup tools
```

```suggestion
      - name: Setup tools
```

他のステップ名と合わせるために大文字始まりであるべき